server/repository/build.rs

```diff
@@ -0,0 +1,18 @@
+use glob::glob;
+use std::fs;
+use std::path::Path;
+
+fn main() {
+    // when migrations are changing mark the template DBs to be recreated
+    for entry in glob("../**/test_output").expect("Failed to read glob pattern") {
+        match entry {
+            Ok(path) => {
+                fs::File::create(Path::new(&path).join("___template_needs_update.marker")).unwrap();
```

Comment:
We could just empty out cache here, might simplify it a little bit. 
Btw can share TEST_OUTPUT_DIR, TEMPLATE_MARKER_FILE constants with [the path attribute](https://doc.rust-lang.org/reference/items/modules.html#the-path-attribute).

Reply:
Thanks did this, i.e. I moved them into a constants file

Reply:
I kept the code in the build.rs minimal because I found it slightly hard to debug and to test the build.rs file (e.g. because of the re-run conditions)... Furthermore, I was also thinking about reusing the marker for Postgres (e.g. Postgres rebuilds template DB when the `test_output` dir doesn't exist OR if there is a marker file in the `test_output` dir)